lib/backend/model/resource.go

```diff
@@ -26,24 +26,90 @@ import (
 	"github.com/projectcalico/libcalico-go/lib/namespace"
```

Comment:
Why alias this as `apiv2` and not just `api`?

Seems like since we're only going to support a single api version, that's the best way to go.

Reply:
Maybe `api` is wrong too - `capi` or something to indicate it's the Calico API vs `kapi` for k8s api? idc.

Reply:
urm iunno - i thought that was done as part of Eriks/Brendans file move

Reply:
yeah, that <didn't> happen during https://github.com/projectcalico/libcalico-go/pull/586. I aliased the import rather than change all the existing `apiv2` namespace references to `v2` or `capiv2`.

Reply:
ok, let's deal with it later.